File: country-picker-dialog-example/src/main/java/io/github/hendraanggrian/countrypickerdialogexample/MainActivity.java
package io.github.hendraanggrian.countrypickerdialogexample;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.Toast;

import io.github.hendraanggrian.countrypickerdialog.Country;
import io.github.hendraanggrian.countrypickerdialog.CountryPickerDialog;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        new CountryPickerDialog.Builder(this)
                .title("Hello!")
                .onPicked(new CountryPickerDialog.OnPickedListener() {
                    @Override
                    public void onPicked(Country country) {
                        Toast.makeText(MainActivity.this, country.getName(), Toast.LENGTH_SHORT).show();
                    }
                }).show();
    }
}

File: demo/custom/src/com/hendraanggrian/appcompat/countrydialog/demo/MainActivity.kt
package com.hendraanggrian.appcompat.countrydialog.demo

import android.os.Bundle
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.hendraanggrian.appcompat.countrydialog.CountryDialog
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        simpleButton.setOnClickListener {
            CountryDialog.Builder(this)
                .setOnSelectedListener { country -> Toast.makeText(this, country.getName(this), Toast.LENGTH_SHORT).show() }
                .show()
        }

        noFlagButton.setOnClickListener {
            CountryDialog.Builder(this)
                .showFlags(false)
                .setOnSelectedListener { country -> Toast.makeText(this, country.getName(this), Toast.LENGTH_SHORT).show() }
                .show()
        }

        dialCodeButton.setOnClickListener {
            CountryDialog.Builder(this)
                .showDialCode(true)
                .setOnSelectedListener { country -> Toast.makeText(this, country.getDialCode(), Toast.LENGTH_SHORT).show() }
                .show()
        }
    }
}